manageiq-operator/pkg/controller/manageiq/manageiq_controller.go

```diff
@@ -439,3 +451,13 @@ func (r *ReconcileManageIQ) createk8sResIfNotExist(cr *miqv1alpha1.ManageIQ, res
 	}
 	return nil
 }
+
+func getSecretKeyValue(client client.Client, nameSpace string, secretName string, keyName string) string {
```

Comment:
I think there's another file with secret methods, where this might belong better.

Reply:
Not that I saw, pod specific helpers (postgres, kafka, etc) to return the default secret name and default secret if not specified.  One possibility is to try to move this to pkg/helpers/miq-components/util.go

Reply:
I don't see a better place for this right now.